frontend/src/app/pages/home-page/home-page.component.scss

```diff
@@ -7,14 +9,80 @@
     display: inline-block;
 }
 
-.mat-fab {
-    height: 200px;
-    width: 200px;
-    margin-left: 1em;
+.account {
+    &-button {
+        @include mat-elevation(4);
+        position: relative;
+        transition: background-color 0.2s ease-in;
+
+        &__options {
+            display: none;
+            transition: all 0.2s ease-in;
+        }
+        &.active {
+            background-color: mat-color($primary, 400);
+            .account-button {
+                &__options {
+                    @include mat-elevation(6);
+                    position: absolute;
+                    top: 105%;
+                    right: 0;
+                    display: flex;
+                    flex-direction: column;
+                    padding: 2em 0.5em;
+                    border-radius: 5px;
+                    background-color: $white-high-emphasis;
+                    color: $black-high-emphasis;
+                }
+                &__link{
+                    padding: 0.5em 2em;
+                    color: $black-high-emphasis;
+                    text-decoration: none;
+                    &:hover {
+                        background-color: gray("200");
+                    }
+                }
+            }
+        }
+    }
 }
 
-.mat-icon {
-    height: inherit;
-    width: inherit;
-    font-size: 5.5em;
+.home {
+    &-welcome {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        flex-wrap: wrap;
+        &__main {
+            flex-basis: 100%;
+            color: $black-medium-emphasis;
+            font-weight: 400;
+        }
+        &__desc {
+            color: $black-8-opacity;
```

Comment:
It will be good to use some darker color here. Current one is pretty non-contrasting.

Reply:
Got it! Definitely you are right :) Kuba has showed me some screenshots and I have probably some contrast issues on my computer   😅